### PR TITLE
Add examples and missing parameters to cluster recovery

### PR DESCRIPTION
**Pull request by kerneljake**
update required rladmin options, font consistency, and general cleanup

### CODE REVIEW

**Comment by bmansheim** [comment]
@kerneljake Thanks for your PRs. They really help to improve the state of the docs.

File: content/rs/administering/troubleshooting/cluster-recovery.md [hunk 1]
```diff
@@ -96,27 +97,31 @@ The command has the following parameters:
...
 
 For each of the other servers that are going to act as RS nodes, run the
-cluster join command from the rladmin CLI prompt in order to join the
+`cluster join` command from the rladmin CLI prompt in order to join the
 node to the recovered cluster, and specify which of the nodes in the old
 cluster this node is going to replace. The command has the following
 parameters:
 
+1. nodes - mandatory. The IP address of a node in the cluster this
+    node is joining. Alternatively, the name parameter can be used.
```
Comment (bmansheim) [inline content/rs/administering/troubleshooting/cluster-recovery.md]: @Paz-Yanover  Who needs to review this addition?

File: content/rs/administering/troubleshooting/cluster-recovery.md [hunk 2]
```diff
@@ -138,9 +143,14 @@ parameters:
     of the old node. You can also change the value of the rack ID by
     providing a different value and using the override_rack_id flag.
 
+Example:
+```src
+$ rladmin cluster join nodes 10.142.0.4 username admin@example.com password mysecret replace_node 2
+```
+
```
Comment (bmansheim) [inline content/rs/administering/troubleshooting/cluster-recovery.md]: @Paz-Yanover Who needs to review this addition?

File: content/rs/administering/troubleshooting/cluster-recovery.md [hunk 3]
```diff
@@ -188,12 +198,11 @@ First, make sure that the recovery path is set correctly on all of the
 nodes in the cluster. If that does not resolve the issues, [contact our
 support team](mailto:support@redislabs.com).
 
-To recover all of the databases, run the rladmin command recover all.
+To recover all of the databases, run the command `rladmin recover all`.
 
-You can also recover the databases one by one, by using the rladmin
-command recover db \<id \| name\>.
+To recover the databases one by one, run the command `rladmin recover db <id|name>`.
```
Comment (bmansheim) [inline content/rs/administering/troubleshooting/cluster-recovery.md]: @Paz-Yanover Who needs to review this addition?

File: content/rs/administering/troubleshooting/cluster-recovery.md [hunk 4]
```diff
@@ -68,19 +68,20 @@ interface
...
 The command has the following parameters:
 
-1. filename - mandatory. The path of the old cluster configuration
+1. filename - mandatory. The full path of the old cluster configuration
     file. The file name is ccs-redis.rdb, and it is located in the /ccs
     directory of the persistent storage drive of the old cluster. This
     file exists on the persistent storage drive of all nodes and
     contains the same exact data, so it does not matter which old node's
     persistence storage you choose to use the file from.
+1. node_uid - mandatory.  The id of the first node, in this case 1.
```
Comment (bmansheim) [inline content/rs/administering/troubleshooting/cluster-recovery.md]: @Paz-Yanover Who needs to review this addition?

Comment (Paz-Yanover) [reply content/rs/administering/troubleshooting/cluster-recovery.md]: @ushachar - who you want to review and approve it from your side?

File: content/rs/administering/troubleshooting/cluster-recovery.md (hunk 1 above)
Comment (Paz-Yanover) [reply content/rs/administering/troubleshooting/cluster-recovery.md]: @ushachar - who you want to review and approve it from your side?

File: content/rs/administering/troubleshooting/cluster-recovery.md [hunk 5]
```diff
@@ -96,27 +97,31 @@ The command has the following parameters:
     set for the node with ID 1, with a new rack ID. Otherwise, the node
     will get the same rack ID as the original node.
 
+Example:
+```src
+$ rladmin cluster recover filename /tmp/persist/ccs/ccs-redis.rdb node_uid 1
+```
```
Comment (Paz-Yanover) [inline content/rs/administering/troubleshooting/cluster-recovery.md]: @ushachar - who you want to review and approve it from your side?

File: content/rs/administering/troubleshooting/cluster-recovery.md (hunk 3 above)
Comment (Paz-Yanover) [reply content/rs/administering/troubleshooting/cluster-recovery.md]: @ushachar - who you want to review and approve it from your side?

**Comment by ushachar** [comment]
@oranagra  - please review for Ben....

**Comment by oranagra** [comment]
I've reviewed the diff, and also read most of the context around it.
everything looks ok to me (i'm not an expert of the recovery process so i can't be sure).
i'm not entirely sure if this review is only for the changes (which are rather minor) or for the whole thing?

**Comment by bmansheim** [comment]
@oranagra If you can at least skim over the page and give your opinion, that would be great because we don't currently have a process for reviewing old docs. If not, we'll release this PR and hope nothing else is wrong with it.

**Comment by oranagra** [comment]
@bmansheim that's what i did.. couldn't find anything that looks wrong to me.

**Pull request merged**